### DISCUSSION TITLE
The ability to make new directories from the browser

### DISCUSSION THREAD

**Discussion by dev1joe** [community/community#70086] [Repositories]
### Select Topic Area

Product Feedback

### Body

I ran into a scenario where I needed to use my phone's browser to add a new directory and upload some some images inside of it. I was only able of uploading these images without putting them into a directory. I think having the ability of creating new directories from the website will be useful.

**Comment by YurinDoctrine** [comment]
You wanted to add a new directory and upload images on your phone's browser. However, you could only upload images without organizing them into a directory. You suggest that adding the ability to create directories directly from the website would be helpful.

**Comment by jamesweber7** [comment]
Creating directories online is unintuitive, but it's actually possible. You can go to your repository -> Add file -> Create new file, and where you enter the name of your file, instead type the directory's name, delimit with a slash (/), then your file's name. Since you want to upload files, you could create the file directory/.gitkeep, upload the files to directory, then delete .gitkeep
